community-fund/COMMUNITY_FUND_POLICY.md

```diff
@@ -0,0 +1,79 @@
+# Cross Project Council Community Fund
+
+## About The Community Fund
+
+The OpenJS Foundation’s community fund supports travel and attendance at events and activities directly related to OpenJS projects, technologies, or goals. We support travel for individuals who are:
+
+* Collaborators
+* Standards body participants
+* Speakers
+
+The fund can cover the costs, in whole or in part, of a participant's hotel, airfare, and some event registrations.
+Applicants should explore employer-sponsored funding options as their first source of financial support.
+Additionally, 25% of the community fund is allocated specifically to support diverse participants.
+Funding decisions are determined by a combination of need and potential impact.
+
+## Community Fund Rules
+
+Valid travel expenses are:
+
+* Coach airfare tickets
+* Accommodations (Dates of the event plus one night prior are eligible for reimbursement)
+* Ground transportation to/from the airport
+
+NOTE: Funds may not be used for miscellaneous travel expenses including food, visa costs, non-airport transportation, baggage fees, etc.
```

Comment:
If/as we're interested in increasing diversity, why are visa costs specifically not allowed? This is a cost almost solely related to potential applicants from underrepresented countries.

Reply:
Big +1, growing up in that situation I know very well how visa costs are a huge deterrent from international travel. Not only visas but also these miscellaneous items can be definitely prohibitive (or a lot of uncertainty) for someone travelling from a country with low purchase power.

Reply:
I did benefit from reimbursing visa cost for my first Node.js summit in Canada + getting an invitation letter from LF for visa process.

Also, I'm not sure if the person requesting, mentions the miscellaneous cost at the time of approval, and it gets approved, even then they can't use it? OR they can't request funds for miscellaneous cost? Wording should be updated.

Reply:
This item requires more discussion. I will create an issue to track this specific question.